apps/rush-lib/src/data/test/repo/rush.json

```diff
@@ -17,6 +17,12 @@
     "sampleEmail": "mrexample@contoso.com"
   },
 
+  "rushHooks": {
+    "postBuild": [
```

Comment:
>postBuild [](start = 5, length = 9)

This should be a little more specific? Is this after each project is built? Is it after "rush build" completes?

Perhaps use a names like "postRushBuild" "preNpmInstall" "postProjectBuild" "preRushPublish" "postRushLink"